grafana-plugin/src/containers/OutgoingWebhookForm/OutgoingWebhookForm.tsx

```diff
@@ -370,46 +275,110 @@ export const OutgoingWebhookForm = observer((props: OutgoingWebhookFormProps) =>
                 </Button>
               )}
               <WithPermissionControlTooltip userAction={UserActions.OutgoingWebhooksWrite}>
-                <Button form={form.name} type="submit" disabled={data.is_legacy}>
-                  {isNewOrCopy ? 'Create' : 'Update'}
+                <Button type="submit" onClick={handleSubmit(onSubmit)} disabled={data.is_legacy}>
+                  Create
                 </Button>
               </WithPermissionControlTooltip>
             </HorizontalGroup>
           </div>
         </div>
-      </>
-    );
-  }
-});
+      </div>
+    </Drawer>
+  );
+};
+
+interface EditWebhookTabsProps {
+  id: OutgoingWebhookFormProps['id'];
+  data: any;
+  action: WebhookFormActionType;
+  onHide: OutgoingWebhookFormProps['onHide'];
+  onUpdate: OutgoingWebhookFormProps['onUpdate'];
+  onDelete: OutgoingWebhookFormProps['onDelete'];
+  onSubmit: (data: Partial<ApiSchemas['Webhook']>) => void;
+  onTemplateEditClick: (params: TemplateParams) => void;
+  preset: OutgoingWebhookPreset;
+}
+
+const EditWebhookTabs = (props: EditWebhookTabsProps) => {
+  const { id, data, action, onHide, onUpdate, onDelete, onSubmit, onTemplateEditClick, preset } = props;
+
+  const history = useHistory();
+
+  const [activeTab, setActiveTab] = useState<string>(
```

Comment:
It's minor but I think the `string` type will be inferred from the default value so we don't need to provide it